ffmpeg/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+link="https://ffmpeg.org/releases/ffmpeg-snapshot.tar.bz2"
```

Comment:
Please use a specific version of ffmpeg, so can have a compatible build. A snapshot version will always be update upstream and it will then result in a different set of features (and maybe command line arguments) at a future time.

You can for the latest stable version shown in the [releases page](https://ffmpeg.org/releases/): `5.1.2`.